configs/guanmai.json

```diff
@@ -1,21 +1,17 @@
 {
   "index_name": "guanmai",
+  "js_render": true,
```

Comment:
Can you avoid to rely on client side rendering please?

Reply:
sorry, can no.I have many document and so I use this way to develop for lesse time.if i use other way that  I will be weast more time.

Reply:
Why is this not recommended this method?

Reply:
if i do not add this,I can't get the exact path.

Reply:
Do you have a clear example please?
Did you run the crawl on your machine?